lambda_functions/alerter/alerter.py

```diff
@@ -32,7 +32,7 @@ def lambda_handler(event, context):
     alert_message = {
         "Summary": f"The website at {url} is unreachable. Healthcheck failure.",
         "Alarm": "Route53 Health Check Failure.",
-        "Description": f"Website is unreachable or not returning the expected response. Check Amazon Cloudwatch Alarms \'{alarm_name}\' and Healthcheck Lambda \'{lambda_name}\'.",
+        "Description": f"Website is unreachable or not returning the expected response. Check Amazon Cloudwatch Alarms '{alarm_name}' and Healthcheck Lambda '{lambda_name}'.",
```

Comment:
You've removed the escaping of the quotes - was this intentional? I wonder if this would cause an issue in the message sent to slack for Healthcheck as the escape characters usually help strings being printed.

Reply:
That's interesting, I haven't edited this file. Will fix now

Reply:
Looks like this was edited by the black code linter

Reply:
FYI black on check-only mode does not flag the escaped characters if you leave them in, but if you run black . it removes them anyway, which is interesting